arduino/smartcar/smartcar.ino

```diff
@@ -15,23 +15,22 @@ WiFiClient net;
 //empty for local host connection
 const char ssid[] = " ";
 const char pass[] = " ";
- 
- 
+const String speedometer = "car/status/odometer/speed";
```

Comment:
I think this line should be removed too. Or changed to "/smartcar/speedometer" and then use it in line 141.